src/JustEat.StatsD/Buffered/BufferExtensions.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Buffers.Text;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace JustEat.StatsD.Buffered
+{
+    internal static class BufferExtensions
```

Comment:
Suggest adding `[EditorBrowsable(EditorBrowsableState.Never)]` to the class to hide it (but not the methods) from Intellisense.

Reply:
Nice tip, never thought of that. I guess it's more important for public types in libraries like this, but still has merit here.